generator/sbpg/targets/resources/sbp_tests_template.rs

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright (C) 2015-2018 Swift Navigation Inc.
+// Contact: Swift Navigation <dev@swiftnav.com>
+//
+// This source is subject to the license found in the file 'LICENSE' which must
+// be be distributed together with this source. All other rights reserved.
+//
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+// This file was auto-generated from (((s.src_filename))) by generate.py. Do not modify by hand!
+
+extern crate sbp;
+use sbp::messages::SBPMessage;
+
+trait AlmostEq {
+    fn almost_eq(self, rhs: Self) -> bool;
+}
+
+impl AlmostEq for f32 {
+    fn almost_eq(self, rhs: Self) -> bool {
+        const ULP: f32 = 5.0;
+        ((self - rhs).abs() <= (std::f32::EPSILON * (self + rhs).abs() * ULP)) || ((self - rhs).abs() < std::f32::MIN)
+    }
+}
+
+impl AlmostEq for f64 {
```

Comment:
Any easy way to not have to repeat this in each test?

Reply:
Moved them into a `common.rs` file which is then used in each test case.